synapse/rest/media/v1/media_repository.py

```diff
@@ -151,7 +151,9 @@ def create_content(self, media_type, upload_name, content, content_length,
             "media_length": content_length,
         }
 
-        yield self._generate_thumbnails(None, media_id, media_id, media_info)
+        yield self._generate_thumbnails(
+            None, media_id, media_id, media_info["media_type"],
```

Comment:
s/media_info["media_type"]/media_type/

media_info now appears to be redundant here